Sources/HaCWebsiteLib/Events/EventServer.swift

```diff
@@ -2,22 +2,25 @@ import Foundation
 
 public struct EventServer {
 
-  static func getEvents() -> [GeneralEvent] {
-    //TODO: source events from database
-    let hypeDuration : Double = 4000000
-    let duration : Double = 4000
-    let workshopEvents : [GeneralEvent] = WorkshopManager.workshops.map{
-      (workshop: Workshop) -> GeneralEvent in
-        return GeneralEvent(title: workshop.title, time:  DateInterval(start: Date(), duration: duration),
-          tagLine: "There's Pizza", color: "purple", hypePeriod: DateInterval(start: Date(), duration: hypeDuration),
-          tags: ["tag1", "tag2"], description: Markdown("Thing"))
-    }.filter{ event in 
-      event.shouldShowAsUpdate
+  private(set) static var events: [GeneralEvent] = []
```

Comment:
Why are we caching? Have you profiled the performance here? If we have a lot of events, wont this potentially be slower?

Reply:
I was just really following the `WorkshopManager` format tbh. Not really sure what is quicker but I kinda assumed a database read would be slow. I may very well be wrong and should probably look into this

Reply:
WorkshopManager was mostly doing this because it has to do a whole bunch of http requests and parsing to update rather than a relatively clean db query. This does feel like premature optimisation probs.

Reply:
Databases are (should) be designed for this kind of thing! I believe if you want to cache database stuff you really need to use something comprehensive like [memcached](https://en.wikipedia.org/wiki/Memcached).

Paging @Pinpickle for an opinion on this.